Sources/TSCUtility/IndexStore.swift

```diff
@@ -14,9 +14,24 @@ import TSCBasic
 public final class IndexStore {
 
     public struct TestCaseClass {
+        public enum TestMethod: Hashable, Comparable {
```

Comment:
Would it be more flexible as a struct instead of an enum? Basically

```swift
public struct TestMethod: Hashable, Comparable {
    var name: String
    var isAsync: Bool
}
```

The way it is currently i feel like ties its basic modeling to this async-or-not distinction, but that’s really just one detail about a test method, and I wonder if it would be more future-proof to allow it to have stored state. I bring it up in part because this type is `public`, thereby raising the importance of modeling it well since it may be harder to change since it’s exported outside its module.

Reply:
I agree that this looks cleaner as a struct.